src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -70,10 +70,28 @@ export const NavSearchBar: React.FC<NavSearchBarProps> = ({ venueId }) => {
     () =>
       relatedVenues
         .flatMap((venue) => venue.rooms ?? [])
-        .filter((room) => room),
+        .filter((room) => {
+          if (isDefined(room.type) && COVERT_ROOM_TYPES.includes(room.type)) {
+            return false;
+          }
+
+          return room.isEnabled;
+        }),
     [relatedVenues]
   );
 
+  const relatedRoomsByTitle = useMemo<Partial<Record<string, Room>>>(
+    () =>
+      relatedRooms.reduce(
+        (relatedRoomsByTitle, room) => ({
+          ...relatedRoomsByTitle,
+          [room.title]: room,
+        }),
+        {}
+      ),
+    [relatedRooms]
+  );
```

Comment:
I suppose this is transforming array into an object. Can we use lodash here?

Reply:
We might be able to.. but why would we? Reduce is a pretty straightforward thing to do.

Reply:
sure, used lodash in a3708be

Reply:
@alexey-petrov see my comments in https://github.com/sparkletown/sparkle/pull/1712#discussion_r665838282 No idea why we used `lodash` when the old code was already using the built in language feature `.reduce` already, and should have continued to be used.

Reply:
@alexey-petrov see my comments in https://github.com/sparkletown/sparkle/pull/1712#discussion_r665838282 No idea why we used `lodash` when the old code was already using the built in language feature `.reduce` already, and should have continued to be used.